0-ad-history-mod-package/simulation/templates/units/athen/infantry_javelineer_b.xml

```diff
@@ -8,6 +8,10 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        Peltasts were a type of Greek light infantry unit originating from Thrace, often fighting as mercenaries for various Greek city-states. They were named after their uniquely crescent-shaped shield (pelte). It was not only smaller than a hoplite's shield but also thinner and lighter due to it being covered in hide rather than bronze. They wore minimal armor and carried a short sword for close combat, in addition to several javelins. Peltasts often used the so-called ankyle, a leather thong attached to the javelin. It was looped over two fingers and slipped off when thrown, extending the leverage and therefore increasing the javelin's momentum and the distance it would travel.
```

Comment:
```suggestion
        Peltasts were a type of Greek light infantry unit originating from Thrace, often fighting as mercenaries for various Greek city-states. They were named after their uniquely crescent-shaped shield (pelte). It was not only smaller than a hoplite's shield, but also thinner and lighter due to it being covered in hide rather than bronze. They wore minimal armor and carried a short sword, in addition to several javelins. Peltasts often used the so-called ankyle, a leather thong attached to the javelin. It was looped over two fingers and slipped off when thrown, extending the leverage and therefore increasing the javelin's momentum and the distance it would travel.
```